packages/logging-operator/overlay/templates/flow.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: logging.banzaicloud.io/v1beta1
```

Comment:
Wouldn't flow and logging.yaml below be conditional on `Values.elasticsearch.enabled` ?